utils/build/virtual_machine/provisions/installer-auto-inject/auto-inject_installer_script.yml

```diff
@@ -0,0 +1,182 @@
+        fi
+        if { [ "$DISTRIBUTION" == "Debian" ] && [ "$release_version" -lt 9 ]; } || \
+          { [ "$DISTRIBUTION" == "Ubuntu" ] && [ "$release_version" -lt 16 ]; }; then
+            # copy with -a to preserve file permissions
+            $sudo_cmd cp -a $apt_usr_share_keyring $apt_trusted_d_keyring
+        fi
+
+        $sudo_cmd DEBIAN_FRONTEND=noninteractive apt-get update
+        $sudo_cmd DD_UPDATER_REGISTRY=${DD_UPDATER_REGISTRY} DD_INSTALLER_OCI_BOOTSTRAP_VERSION=${DD_PIPELINE_ID} DD_UPDATER_REGISTRY_AUTH=${DD_UPDATER_REGISTRY_AUTH} apt-get install -y --force-yes datadog-installer
+
+        # Only for systemd
+        exit_status=0
+        $sudo_cmd systemctl status datadog-installer || exit_status=$?
+        if [ $exit_status -ne 4 ]; then # Status 4 means the unit does not exist
+            $sudo_cmd systemctl daemon-reload
+            $sudo_cmd systemctl stop datadog-installer
+        fi
+        # Add packages
+        for pkg in ${PACKAGES[@]}; do
+            $sudo_cmd $INSTALLER_BIN bootstrap --url "${OCI_URL_PREFIX}${pkg}"
+        done
+        if [ $exit_status -ne 4 ]; then # Status 4 means the unit does not exist
+            $sudo_cmd systemctl start datadog-installer
+        fi
+    elif [ "${OS}" = "RedHat" ]; then
+        yum_url="yumtesting.datad0g.com/testing"
+        yum_repo_version="${DD_PIPELINE_ID}-i7/7"
+
+        RPM_GPG_KEYS=("DATADOG_RPM_KEY_CURRENT.public" "DATADOG_RPM_KEY_B01082D3.public" "DATADOG_RPM_KEY_FD4BF915.public" "DATADOG_RPM_KEY_E09422B3.public")
+        separator='\n       '
+        for key_path in "${RPM_GPG_KEYS[@]}"; do
+            gpgkeys="${gpgkeys:+"${gpgkeys}${separator}"}https://${keys_url}/${key_path}"
+        done
+        $sudo_cmd sh -c "echo -e '[datadog]\nname = Datadog, Inc.\nbaseurl = https://${yum_url}/${yum_repo_version}/${ARCH}/\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\npriority=1\ngpgkey=${gpgkeys}' > /etc/yum.repos.d/datadog.repo"
+        $sudo_cmd yum -y clean metadata
+        $sudo_cmd DD_UPDATER_REGISTRY=${DD_UPDATER_REGISTRY} DD_INSTALLER_OCI_BOOTSTRAP_VERSION=${DD_PIPELINE_ID} DD_UPDATER_REGISTRY_AUTH=${DD_UPDATER_REGISTRY_AUTH} yum -y install datadog-installer
+    elif [ "${OS}" = "SUSE" ]; then
+        yum_url="yumtesting.datad0g.com/suse/testing"
+        yum_repo_version="${DD_PIPELINE_ID}-i7/7"
+
+        RPM_GPG_KEYS=("DATADOG_RPM_KEY_CURRENT.public" "DATADOG_RPM_KEY_B01082D3.public" "DATADOG_RPM_KEY_FD4BF915.public" "DATADOG_RPM_KEY_E09422B3.public")
+        separator='\n       '
+        for key_path in "${RPM_GPG_KEYS[@]}"; do
+            gpgkeys="${gpgkeys:+"${gpgkeys}${separator}"}https://${keys_url}/${key_path}"
+        done
+        $sudo_cmd sh -c "echo -e '[datadog]\nname = Datadog, Inc.\nbaseurl = https://${yum_url}/${yum_repo_version}/${ARCH}/\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\npriority=1\ngpgkey=${gpgkeys}' > /etc/zypp/repos.d/datadog.repo"
+        $sudo_cmd zypper -n --gpg-auto-import-keys refresh
```

Comment:
you can replace above lines by running  
DD_REPO_URL=datad0g.com DD_AGENT_DIST_CHANNEL=beta DD_NO_AGENT_INSTALL=true bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"